README.rst

```diff
@@ -61,7 +61,7 @@ called through its public C API using the `ctypes <https://docs.python.org/2/lib
 **nimi-python** supports all the Operating Systems supported by the underlying driver.
 
 **nimi-python** follows `Python Software Foundation <https://devguide.python.org/#status-of-python-branches>`_ support policy for different versions. At
-this time this includes Python 3.7 and above using CPython.
```

Comment:
This sentence is a maintenance burden.
If we have a Python list of supported Python versions, we should use that. If we don't, I'm ok removing this incomplete redundant information.

Reply:
We don't have it stored in metadata. Removing it.